base/linalg/blas.jl

```diff
@@ -112,6 +112,29 @@ function set_num_threads(n::Integer)
     return nothing
 end
 
+"""
+    get_num_threads(n)
+
+Get the number of threads the BLAS library can use.
+"""
+function get_num_threads()
```

Comment:
the vendor() function is in a try, so I copied that technique and return nothing if our known functions fail. is this too much of a hammer?